src/components/Catalog.vue

```diff
@@ -135,13 +135,22 @@
                 <!-- TODO row-details w/ additional metadata + map -->
               </b-table>
               <b-pagination
-                v-if="itemCount > itemsPerPage"
+                v-if="!hasExternalPagination"
```

Comment:
These two conditions can be combined

Reply:
yes, then I have to use another `v-if` instead of `v-else` on the `<ul>` under, maybe it would be clearer this way (I'm not a huge fan of `v-else` in big templates)

Reply:
Wouldn't the following work? That seems to be the cleanest way for me. The v-show clutters the DOM.

`<b-pagination v-if="!hasExternalPagination && itemCount > itemsPerPage"`

`<ul v-else class="pagination" v-if="hasExternalPagination ">`